### ISSUE TITLE
Filled in fields not registered by page

### ISSUE THREAD

**Issue by davebush** [sblask/webextension-simple-form-fill#34]
### New Issue Checklist (tick off like this: `[x]`)

- [x ] I checked existing issues (open and closed) for possible duplicates
- [ x] I can reproduce the problem on the latest stable version (not nightly!)

### What is your browser?

- [ x] Firefox
- [ ] Chrome

### What is your operating system?

- [ ] Linux
- [ ] Mac
- [ x] Windows

### Description (please include examples/screenshots where applicable)
On increasingly many site after the field has been filled in, the page doesn't notice the update and just goes with the 1-2 characters actually typed. I assume this is some sort of character based event rather than field based - but I'm not a browser expert.

**Comment by sblask** [comment]
I have seen your problem but only rarely. I would usually type whitespace backspace after filling and it would fix the problem. I might be able to fix it, but I would need an example page from you as I don't have the problem on the pages I visit regularly.

**Comment by davebush** [comment]
If you go to
https://new.myhermes.co.uk/track.html#/parcel/0587181618107290/details
and type the initial letter of an autofill post code (e.g. M20 6ST) and submit - it only takes the 'm'

Dave

**Comment by davebush** [comment]
> 
> 
> I have seen your problem but only rarely. I would usually type whitespace backspace after filling and it would fix the problem. I might be able to fix it, but I would need an example page from you as I don't have the problem on the pages I visit regularly.

The best example I've seen is here: https://www.bmstores.co.uk/stores Autofill the location, then just click outside the field !!

**Comment by sblask** [comment]
I looked into this for a bit now and I cannot reproduce the problem on myhermes, but I can on bmstores. However, the problem is not what I expected, it's a third party library used on the website that explicitly resets the field value. I tried a few things, but nothing fixes the problem. If you have other websites with this kind of problem, I can look into those, otherwise I don't think I can do anything.
